ix-dev/community/lyrion-music-server/templates/test_values/basic-values.yaml

```diff
@@ -0,0 +1,44 @@
+resources:
+  limits:
+    cpus: 2.0
+    memory: 4096
+
+lms:
+  additional_envs: []
+
+network:
+  web_port:
+    port_number: 9000
+    container_port: 9000
+    bind_mode: published
+  cli_port:
+    port_number: 9090
+    container_port: 9090
+    bind_mode: published
+
+run_as:
+  user: 568
+  group: 568
+
+ix_volumes:
+  config: /mnt/storage/lms/config
+  music: /mnt/storage/lms/music
+  playlist: /mnt/storage/lms/playlist
```

Comment:
```suggestion
  config: /opt/tests/mnt/storage/lms/config
  music: /opt/tests/mnt/storage/lms/music
  playlist: /opt/tests/mnt/storage/lms/playlist
```

Reply:
Mostly to keep it consistent with the rest.
I dont want someone to accidentaly run the tests and files in their /mnt/storage are getting written/overwritten

Reply:
Is there a standard way to get `/opt/tests` set up on a TrueNAS Scale system?  `/opt` is on the boot-pool, which is read-only.  I've seen references to remounting it r/w, but also warnings that that's not supported (might lead Support to ignoring questions).

Reply:
Those paths are picked for local development. ie your laptop/pc
Specifically the /opt/tests are picked because macOS whitelists those by default
(https://github.com/truenas/apps/blob/master/CONTRIBUTIONS.md#appyaml)

Reply:
Yep, I saw that.  But, I'm not doing this work on my laptop, I'm doing it on my TrueNAS scale system.  Those paths are particularly problematic on a Scale box.

I'd happily just carve off a data set from my storage pool, but I don't see any supported way to mount it on /opt/tests.  Trying to to go wandering off into the forest if I can help it.

Any Scale suggestions?

Reply:
Developing apps on scale is not really easy. Well at least I don't find it easy.
Changing the paths during dev is fine. But I think most people will use their computers to do dev.
Hence insisting on keeping the paths under /opt/tests/